File: client/android/Contract/Attachment/AttachmentUploadHandoffPayload.kt
package app.mobiling.client.contract.attachment

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class AttachmentUploadHandoffPayload(
    val uploadUrl: String,
    val method: String,
    val fieldName: String,
    val handoffMode: String,
    val payloadText: String,
)

File: client/ios/Contract/Attachment/AttachmentUploadHandoffPayload.swift
import Foundation

public struct AttachmentUploadHandoffPayload: Sendable {
    public let uploadUrl: String
    public let method: String
    public let fieldName: String
    public let handoffMode: String
    public let payloadText: String?

    public init(uploadUrl: String, method: String, fieldName: String, handoffMode: String, payloadText: String?) {
        self.uploadUrl = uploadUrl
        self.method = method
        self.fieldName = fieldName
        self.handoffMode = handoffMode
        self.payloadText = payloadText
    }
}
